plugins/ksp/src/org/jetbrains/kotlin/ksp/processor/TypeComparisonProcessor.kt

```diff
@@ -49,6 +49,13 @@ open class TypeComparisonProcessor : AbstractTestProcessor() {
 class TypeCollector : KSTopDownVisitor<MutableCollection<KSType>, Unit>() {
     override fun defaultHandler(node: KSNode, data: MutableCollection<KSType>) = Unit
 
+    // TODO: fix test cases using type collector?
```

Comment:
Yes, please update them rather than work around here. Or extend TypeCollector to TypeCollectorNoPropertyAccessor and update the processors.